src/cdi/CDI.hh

```diff
@@ -201,7 +201,7 @@ static double Planck2Rosseland(double const freq, double const exp_freq) {
   double factor;
```

Comment:
I know you didn't change this part of the code, but please initialize this variable at construction (`double factor(42.0);`)